l2.wasm.go

```diff
@@ -0,0 +1,56 @@
+//go:build wasm
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"syscall/js"
+
+	controller "github.com/HexmosTech/lama2/controller"
+)
+
+var version string
+
+func init() {
+	// Initialize version if not set
+	if len(version) == 0 {
+		version = "vUnset"
+	}
+}
+
+func main() {
+	// Set up the WebAssembly function
+	js.Global().Set("lama2Wasm", js.FuncOf(lama2Wasm))
+	js.Global().Set("goWebRequestFunc", wasmLamaPromise())
+	select {}
+}
+
+func lama2Wasm(this js.Value, i []js.Value) interface{} {
+	go func() {
+		controller.ProcessWasmInput("GET\nhttps://httpbin.org/get")
```

Comment:
Is this for processing the Default request?

Reply:
That function was for testing purpose , removed